src/modules/book/components/book-comment-section.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from 'react';
+
+import BookComment from '@/modules/book/components/book-comment';
+import { Comment } from '@/types/comment';
+
+interface BookCommentSectionProps {
+  comments: Comment[];
+}
+const BookCommentSection: FC<BookCommentSectionProps> = ({ comments }) => {
+  return (
+    <div className="space-y-4">
+      {comments.map((comment) => (
+        <BookComment
+          key={comment.id}
+          id={comment.id}
+          userName={comment.userName}
+          content={comment.content}
+          likesNumber={comment.likesNumber}
+          dislikesNumber={comment.dislikesNumber}
+          rating={comment.rating}
```

Comment:
pass a full object as the props instead of many props with comment properties, and then in BookComment you can enumerate properties like there `const { id, userName } = comment`